src/main/java/seedu/address/logic/parser/AddApplicantCommandParser.java

```diff
@@ -55,4 +70,21 @@ public AddApplicantCommand parse(String args) throws ParseException {
     private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
+
+    /**
+     * Formats the error message for missing prefixes to be displayed to the user.
+     */
+    private String formatMissingPrefixesErrorMessage(ArrayList<String> missingPrefixes) {
+        StringBuilder errorMessage = new StringBuilder("Missing fields: ");
+        int len = missingPrefixes.size();
+        for (int i = 0; i < len; i++) {
+            errorMessage.append(missingPrefixes.get(i));
+            if (i >= len - 1) {
+                break;
+            }
+            errorMessage.append(", ");
```

Comment:
You can make use of StringUtils.join(arrayList, ",") to combine the final list instead